### PR TITLE
Support execution of multiple command files as described in #2398 

### LINKED ISSUE #2398: SCDF Shell execute  multiple command files
From the scdf shell documentation, it looks like multiple files execution ARE supported
````
--spring.shell.commandFile=...  //Data Flow Shell executes commands read from the file(s) and then exits.
````
but `spring.shell.commandFile` property maps to single file.
Please support execution of multiple files passed  via  property (comma separated string  or list )

### PR DESCRIPTION
**Pull request by jvmlet**
integration tests and docs are also here

Resolves #2398.

### CODE REVIEW

File: spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/ShellCommandLineParser.java [hunk 1]
```diff
@@ -57,12 +58,13 @@
 	public CommandLine parse(ShellProperties shellProperties, String[] applicationArguments) {
 		List<String> commands = new ArrayList<String>();
 		if (shellProperties.getCommandFile() != null) {
```
[the remark this answers is not in the document]
Comment (jvmlet) [reply spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/ShellCommandLineParser.java]: Thanks, @tzolov  for your comments.
Sure, I will change the bound type to `List<String>` and let converter to do it's job, just want to make sure that the passed argument stays the same (to be backward compatible) : 
`--spring.shell.commandFile=file1.txt,file2.txt`  
and not 
`--spring.shell.commandFile[0]=file1.txt` and `--spring.shell.commandFile[1]=file2.txt`
 (which is not backward compatible and will break users' scripts). 
Please confirm

[the remark this answers is not in the document]
Comment (tzolov) [reply spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/ShellCommandLineParser.java]: Yep, it should be backward compatible. In fact i believe that your existing tests will/should pass without changes. 
- Cheers

[the remark this answers is not in the document]
Comment (jvmlet) [reply spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/ShellCommandLineParser.java]: @tzolov , I've incorporated the changes and added another test to validate the order of commands is preserved.
The build fails on `Spring Cloud Data Flow Docs for Classic mode`  module with the same error  as PR #2421.
Please review.

[the remark this answers is not in the document]
Comment (tzolov) [reply spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/ShellCommandLineParser.java]: Thanks @jvmlet , I will review it now. 
Btw what name should i use for the `@authors` doc? `Furer Alexander`?

[the remark this answers is not in the document]
Comment (jvmlet) [reply spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/ShellCommandLineParser.java]: Yes, Furer Alexander
Thanks

**Comment by tzolov** [comment]
- some minor fixes applied
- rebased to 8a86606c995e1bd4c4633184f6dd5d41ac069995
@jvmlet thanks for your contribution!

**Pull request closed without merging**